e2etest/newe2e_task_resourcemanagement.go

```diff
@@ -220,3 +220,14 @@ func ValidateErrorOutput(a Asserter, stdout AzCopyStdout, errorMsg string) {
 	fmt.Println(stdout.String())
 	a.Error("expected error message not found in azcopy output")
 }
+
+func ValidateStatsReturned(a Asserter, stdout AzCopyStdout) {
```

Comment:
I notice us putting a lot of these validate* functions in `newe2e_task_resourcemanagement.go`-- It might make sense to just have a `newe2e_validation.go`...

Reply:
yeah I can create a new file for all these validation functions

Reply:
done